blocks/api/serializer.js

```diff
@@ -178,6 +178,10 @@ export function getCommentDelimitedContent( blockName, attributes, content ) {
 		? serializeAttributes( attributes ) + ' '
 		: '';
 
+	if ( blockName.startsWith( 'core/' ) ) {
+		blockName = blockName.substr( 5 );
+	}
```

Comment:
would you be happy using a style that doesn't mutate our variables? that is, instead of changing mid-function we start with a parameter and acknowledge that it needs processing via an appropriate name, then binding once the final value for that name?

```js
getCommentDelimitedContent( rawBlockName, attributes, content ) {
	…
	// strip core blocks of their namespace prefix
	const blockName = rawBlockName.startsWith( 'core/' )
		? rawBlockName.substr( 5 )
		: rawBlockName;
	…
}
```

also, there's nothing wrong with it, though it seems like I'm not as used to seeing `String.prototype.substr()` as frequently as `String.prototype.slice()` - and I think the behaviors are the same when only one parameter is supplied. thoughts?

Reply:
Yah, I'm cool with these changes.

I probably defaulted to `substr()` because of PHP, but `slice()` is clearly the default option in the codebase.

Reply:
Thanks @pento - I'm a bit intimidated asking for a style change, so thank for being so gracious with it!